### PR TITLE
doc: matter: add known issue related to Android CHIPTool

### PR DESCRIPTION
**Pull request by Damian-Nordic**
Add a known issue for NCS 1.7.0 about Android CHIPTool not
being able to communicate with a Matter device after the
device reboots.

Signed-off-by: Damian Krolik <damian.krolik@nordicsemi.no>

### CODE REVIEW

**Review by greg-fer — CHANGES_REQUESTED** [review]
Isn't this an issue related to the external Android CHIPtool and not to how NCS is working? Should we really document it in NCS known issues? Also, is there a workaround, like "Use Python CHIP Controller instead."?

**Comment by Damian-Nordic** [comment]
> Isn't this an issue related to the external Android CHIPtool and not to how NCS is working? Should we really document it in NCS known issues? Also, is there a workaround, like "Use Python CHIP Controller instead."?

It is, but we use Android CHIPTool in our docs and we used to add known issues related to Android CHIPtool here as well. So you're saying we should not and there's no alternative way to communicate known issues related to testing tools?

We refer to both Android CHIPTool and Python CHIP Controller in our docs (for different scenarios and testing environments), so if a customer wants to use Android CHIPTool I'm not sure recommending the Python CHIP Controller will make them happy. Also, one doesn't need to reboot the device to test Matter so the issue is related with a very specific use case.

**Review by greg-fer — APPROVED** [review]

**Review by greg-fer — CHANGES_REQUESTED** [review]
Sorry, please DNM until we have a PR for known issues review.

File: doc/nrf/known_issues.rst [hunk 1]
```diff
@@ -489,6 +489,14 @@ KRKNWK-6073: Potential delay during FOTA
 Matter (Project CHIP)
 =====================
 
+.. rst-class:: v1-7-0
```
Comment (greg-fer) [inline doc/nrf/known_issues.rst:492]:
```suggestion
.. rst-class:: v1-6-1
```
Please do not add tag for next release before it took place. The current version is still 1.6.1, so please use `v1-6-1` or wait till after RC1 and the PR that adds 1.7.0.

**Comment by greg-fer** [comment]
Please merge this to #5475 or add it as comment there and close this PR.

**Review by greg-fer — APPROVED** [review]

**Comment by Damian-Nordic** [comment]
@greg-fer Created a PR to your branch: https://github.com/greg-fer/fw-nrfconnect-nrf-1/pull/3. Closing this one.

**Pull request closed without merging**